msa_sdk/msa_api.py

```diff
@@ -21,7 +22,13 @@ def host_port():
     Hostname and Port
 
     """
-    return('localhost', '8480')
+    host = 'localhost'
+    if ('UBIQUBE_MSA_HOST' in os.environ):
+        host = os.environ['UBIQUBE_MSA_HOST']
```

Comment:
```suggestion
    if ('UBIQUBE_MSA_HOST' in os.environ):
        host = os.environ['UBIQUBE_MSA_HOST']
    else
        host = 'localhost'
```

Reply:
or:
```
host = os.environ.get('UBIQUBE_MSA_HOST') or "localhost"
```

Reply:
@fambon ,
Thank you, it's exactly how I want to write those things !